spec/nine_one_one/pager_duty_service_spec.rb

```diff
@@ -13,15 +13,16 @@
       use_vcr_cassette 'pager_duty_trigger_event_success', match_requests_on: [:body]
 
       it 'calls pager duty service with proper json' do
-        pager_service.trigger_event('Incident description', source: 'source info', details_hash: details_hash, severity: severity)
+        pager_service.trigger_event('Incident description', source: 'source info', details_hash: details_hash,
```

Comment:
If you started using `message` I'd get rid of the `'Incident description'` here as well.